Gemfile.lock

```diff
@@ -824,4 +824,4 @@ DEPENDENCIES
   youtube-dl.rb
 
 BUNDLED WITH
-   1.14.5
+   1.14.6
```

Comment:
I personally just track latest bundler on my development machine (which is now `1.14.6`), even though our infrastructure is locked to `1.10.6`.

As I understand it, our current state is that we officially _recommend_ the same version as our infra (`1.10.6`) for full compatibility, but don't _require_ everyone lock to that version (since newer versions are all backwards-compatible).

The `BUNDLED_WITH` line is an annoying artifact (and causes a notice to appear on earlier versions of Bundler), but I'd rather live with that than have to keep Bundler locked to `1.10.6` on my dev workstation, since newer versions bring performance improvements that make the tool easier to use. At the same time, I'd prefer to keep infra more conservatively locked to `1.10.6` rather than update more often, because any updates bring a (tiny) risk of regressions that could affect production (would take effort to test/verify on each update). So the status quo isn't perfect but I think it's still the best compromise I can think of.